services/connectivity/messaging/src/main/java/org/eclipse/ditto/services/connectivity/messaging/amqp/AmqpConsumerActor.java

```diff
@@ -366,7 +366,7 @@ private void acknowledge(final JmsMessage message, final boolean isSuccess, fina
                 ackType = ACCEPTED;
                 ackTypeName = "accepted";
             } else {
-                ackType = redeliver ? MODIFIED_FAILED : MODIFIED_FAILED_UNDELIVERABLE;
+                ackType = redeliver ? MODIFIED_FAILED : REJECTED;
                 ackTypeName = redeliver ? "modified[delivery-failed]" : "modified[delivery-failed,undeliverable-here]";
```

Comment:
@yufei-cai please also adjust the `ackTypeName` accordingly to "rejected" - this is used only for logging but it would be confusing if that would differ.